### PR TITLE
chore: rewrite/reorg access control docs

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: admin/access-control/manage.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Authentication management
+description: Learn how to manage Coder authentication.
+---
+
+By default, Coder enables **built-in authentication**, though you can change
+this if desired.
+
+To do so, go to **Manage** > **Users**. Find the user whose authentication type
```
Comment (jsjoeio) [inline admin/access-control/manage.md:9]: nit: here it looks like we're bolding the directions "**Manage** > **Users**"

but below we don't when we say "Under Admin > Manage >
Authentication"

Is this intentional?

File: admin/access-control/users/index.md [hunk 2]
```diff
@@ -26,6 +25,18 @@ which they can use with their email to access their new account. For increased
...
+## Changing a user's role
+
+Coder comes with built-in [user roles](user-roles.md) that define what actions a
+user can take in the deployment.
+
+By default, all new users are assigned the **Member** role. These users can be
+upgraded to **Auditor** or **Site Manager** by another user with administrative
+privileges.
+
+To change a user's role, go to **Manage** > **Users**. Find the user and use the
```
Comment (jsjoeio) [inline admin/access-control/users/index.md:37]: same as above: is the rule to always bold directions we give to the users?

**Review by jsjoeio — APPROVED** [review]
Nice work @khorne3 🎉

File: admin/access-control/manage.md (hunk 1 above)
Comment (khorne3) [reply admin/access-control/manage.md:9]: Unintentional. I try to bold UI instructions to help people skim and find stuff

**Pull request merged**